x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S=${WORKDIR}/rofi-${PV}+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun test +windowmode"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango[X]
+	x11-libs/startup-notification
+	x11-libs/xcb-util
+	x11-libs/xcb-util-cursor
+	x11-libs/xcb-util-wm
+	x11-misc/xkeyboard-config
+	dev-libs/wayland
+	dev-libs/wayland-protocols
+"
+DEPEND="
```

Comment:
This version of rofi can't be installed together with ::gentoos x11-misc/rofi, can it? Because in this case we should add a blocker for it 
```
 !x11-misc/rofi
```
yup:
```
 * Detected file collision(s):
 *
 * 	/usr/lib64/pkgconfig/rofi.pc
 * 	/usr/share/rofi/themes/iggy.jpg
 * 	/usr/share/rofi/themes/iggy.rasi
 * 	/usr/share/rofi/themes/fancy.rasi
 * 	/usr/share/rofi/themes/solarized_alternate.rasi
 * 	/usr/share/rofi/themes/solarized.rasi
 * 	/usr/share/rofi/themes/sidebar-v2.rasi
 * 	/usr/share/rofi/themes/sidebar.rasi
 * 	/usr/share/rofi/themes/purple.rasi
 * 	/usr/share/rofi/themes/paper-float.rasi
 * 	/usr/share/rofi/themes/lb.rasi
 * 	/usr/share/rofi/themes/gruvbox-light.rasi
 * 	/usr/share/rofi/themes/gruvbox-light-soft.rasi
 * 	/usr/share/rofi/themes/gruvbox-light-hard.rasi
 * 	/usr/share/rofi/themes/gruvbox-dark.rasi
 * 	/usr/share/rofi/themes/gruvbox-dark-soft.rasi
 * 	/usr/share/rofi/themes/gruvbox-dark-hard.rasi
 * 	/usr/share/rofi/themes/gruvbox-common.rasi
 * 	/usr/share/rofi/themes/glue_pro_blue.rasi
 * 	/usr/share/rofi/themes/docu.rasi
 * 	/usr/share/rofi/themes/dmenu.rasi
 * 	/usr/share/rofi/themes/c64.rasi
 * 	/usr/share/rofi/themes/blue.rasi
 * 	/usr/share/rofi/themes/arthur.rasi
 * 	/usr/share/rofi/themes/android_notification.rasi
 * 	/usr/share/rofi/themes/Paper.rasi
 * 	/usr/share/rofi/themes/Monokai.rasi
 * 	/usr/share/rofi/themes/Indego.rasi
 * 	/usr/share/rofi/themes/DarkBlue.rasi
 * 	/usr/share/rofi/themes/Arc-Dark.rasi
 * 	/usr/share/rofi/themes/Arc.rasi
 * 	/usr/share/rofi/themes/Adapta-Nokto.rasi
 * 	/usr/share/applications/rofi.desktop
 * 	/usr/share/applications/rofi-theme-selector.desktop
 * 	/usr/share/man/man5/rofi-dmenu.5.bz2
 * 	/usr/share/man/man5/rofi-debugging.5.bz2
 * 	/usr/share/man/man5/rofi-keys.5.bz2
 * 	/usr/share/man/man5/rofi-script.5.bz2
 * 	/usr/share/man/man5/rofi-theme.5.bz2
 * 	/usr/share/man/man1/rofi.1.bz2
 * 	/usr/share/man/man1/rofi-sensible-terminal.1.bz2
 * 	/usr/share/man/man1/rofi-theme-selector.1.bz2
 * 	/usr/include/rofi/rofi-icon-fetcher.h
 * 	/usr/include/rofi/rofi-types.h
 * 	/usr/include/rofi/helper.h
 * 	/usr/include/rofi/mode-private.h
 * 	/usr/include/rofi/mode.h
 * 	/usr/bin/rofi-theme-selector
 * 	/usr/bin/rofi-sensible-terminal
 * 	/usr/bin/rofi
 *
```